src/Microsoft.Extensions.FileProviders.Abstractions/project.json

```diff
@@ -3,7 +3,7 @@
   "buildOptions": {
     "warningsAsErrors": true,
     "keyFile": "../../tools/Key.snk",
-    "xmlDoc": true
+    "xmlDoc": false
```

Comment:
To be clear, this suppresses xml doc generation completely. If you just want to disable the warning, use "nowarn"